### PR TITLE
Change from Travis to Circle, modify origami.json

### PR DESCRIPTION
**Pull request by j-mes**
- Check for correct support link
- Change from Travis to Circle CI
- Remove travis.yml
- Add circle.yml
- Passed OBT verify check
- Passed OBT runServer check

Also added a semi-colon to main.js at line 228, col 29 as eslint threw out an error for that.

### CODE REVIEW

**Comment by j-mes** [comment]
All checks have passed.

File: origami.json [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
+	"support": "https://github.com/Financial-Times/o-expander/issues",
+	"demosDefaults": {
+		"js": "demos/src/demo.js",
+		"sass": "demos/src/demo.scss"
+	},
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-expander"
+	},
+	"demos": [
+		{
```
Comment (onishiweb) [inline origami.json]: Be good to get the indenting consistent. Either drop the `{` back an indent, or indent the contents one more tab.

File: origami.json [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
-            "expanded": false,
-            "description": "More advanced configurations of the module"
-        }
-    ]
+	"description": "Accessible, content-aware widget for expanding and collapsing content",
+	"origamiVersion": 1,
+	"origamiType": "module",
+	"browserFeatures": {
+		"required": [
+		"classlist",
```
Comment (onishiweb) [inline origami.json]: These should be indented one more tab

**Comment by onishiweb** [comment]
Couple of indenting comments but other than that looks good.

File: origami.json (hunk 1 above)
Comment (j-mes) [inline origami.json]: It should be fixed now.

**Pull request merged**